src/main/java/org/vivoweb/harvester/util/FileAide.java

```diff
@@ -300,15 +301,35 @@ public static OutputStream getOutputStream(String path, boolean append) throws I
 	 * @throws IOException error resolving path
 	 */
 	public static Set<String> getNonHiddenChildren(String path) throws IOException {
-		Set<String> allFileListing = new HashSet<String>();
-		for(FileObject file : getFileObject(path).findFiles(Selectors.SELECT_CHILDREN)) {
-			if(!file.isHidden() && (file.getType() == FileType.FILE)) {
+		Set<String> allFileListing = new HashSet<>();
+
+		// Process sub-directories
+		for (FileObject file : getFileObject(path).findFiles(Selectors.SELECT_CHILDREN)) {
```

Comment:
It would be easier to read and maintain without nesting when a guard class continue is possible.

e.g.
```
for (FileObject file : getFileObject(path).findFiles(Selectors.SELECT_CHILDREN)) {
	if (file.isHidden() || file.getType() != FileType.FOLDER) {
		continue;
	}
	// Move all files in children directories to the parent folder
	for (FileObject childFile : file.findFiles(Selectors.SELECT_CHILDREN)) {
		if (childFile.isHidden() || childFile.getType() != FileType.FILE) {
			continue;
		}
		FileObject parentFolder = file.getParent();
		if (parentFolder != null) {
			FileObject targetFile = parentFolder.resolveFile(childFile.getName().getBaseName());
			childFile.moveTo(targetFile);
		}
	}
}
```

Reply:
Furthermore, probably should create a method for the redundant condition.

i.e.
```
public boolean isValidFile(FileObject file, FileType type) {
    return !file.isHidden() && file.getType() == type;
}
```

Reply:
Done and done :smile: Thank you for the review!